app/repository/RaceGroupRepositoryImpl.java

```diff
@@ -1,20 +1,21 @@
 package repository;
 
-import models.Judgment;
+import com.fasterxml.jackson.databind.JsonNode;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.fasterxml.jackson.databind.JsonNode'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)